outputs.tf

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project" {
+  description = "The project where the cloud endpoint was created."
+  value       = "${local.project}"
+}
+
+output "external_ip" {
+  description = "The value of the external IP the endpoint points to."
+  value       = "${local.external_ip}"
+}
+
+output "endpoint" {
+  description = "The name of the DNS record conventional to the Cloud Endpoints format of: NAME.endpoints.PROJECT.cloud.goog. Not dependent on google_endpoints_service resource."
```

Comment:
Is it intentional to make this non-dependent? Might be easier logic if it were.

Reply:
So in the ACME example, I wanted to use the generated endpoint DNS name as [part of the instance metadata](https://github.com/danisla/terraform-google-endpoints-dns/blob/initial-import/examples/gce-acme/main.tf#L84), this output lets you do that without creating a dependency cycle.

Reply:
the service_name variable is the dependent variable. Should we rename them?

Reply:
👍 The values are the same. I’ll update it.